test/correctness/fast_arctan.cpp

```diff
@@ -0,0 +1,87 @@
+#include "Halide.h"
+
+#ifndef M_PI
+#define M_PI 3.14159265358979310000
+#endif
+
+using namespace Halide;
+
+int main(int argc, char **argv) {
+    Target target = get_jit_target_from_environment();
+
+    struct Prec {
+        ApproximationPrecision precision;
+        float epsilon;
+    } precisions_to_test[] = {
+        {ApproximationPrecision::MAE_1e_2, 1e-2f},
+        {ApproximationPrecision::MAE_1e_3, 1e-3f},
+        {ApproximationPrecision::MAE_1e_4, 1e-4f},
+        {ApproximationPrecision::MAE_1e_5, 1e-5f},
+        {ApproximationPrecision::MAE_1e_6, 1e-6f}};
+
+    for (Prec precision : precisions_to_test) {
+        fprintf(stderr, "\nTesting for precision %e...\n", precision.epsilon);
```

Comment:
Nit: this isn't an error, so it should go to stdout (also more below), only send error-or-warning output to stderr

Reply:
Oh good point. I was confused: I thought you _had_ to send stuff to stderr to make the tests work. I thought stdout was reserved for sending "Success!".